packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -496,6 +508,11 @@ export default defineComponent({
   min-width: 5rem !important;
   display: flex;
   justify-content: flex-end;
+  .expiration-date-hint {
+    display: flex;
+    align-items: center;
```

Comment:
we have those too

oc-display-flex 
and oc-align... idk you need to look it up lol

Reply:
we could use oc-text-muted

Reply:
changed, just not sure why using classes is better than using styling (with the correct vars)

Reply:
but both fine with me

Reply:
because you can see it directly in the template and you don't need to scroll